lint-configs/symlink-cpp-lint-configs.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Exit on any error
+set -e
+
+# Error on undefined variable
+set -u
+
+repo_dir="$(git rev-parse --show-toplevel)"
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Symlinks the given config file to the repo's root.
+#
+# @param $1 Path to the config file in the repo.
+function symlink_config () {
+    config_file_path="$1"
+
+    repo_dir="$(git rev-parse --show-toplevel)"
```

Comment:
Is this redundant with early assignment?

Reply:
Good catch